### ISSUE TITLE
chamssssssss

### ISSUE THREAD

**Issue by gayblackirony** [tf2cheater2013/Fedoraware#143]
**Fedoraware Version Date:idfk latest one (april 5th)**  

## Config used:
https://files.catbox.moe/juifg6.fed

## Description:  
Some weird shit happens when i turn on fresnel for enemies

## Steps to reproduce:  
- [yes] I am able to consistently reproduce this error
Steps are in the vid (https://streamable.com/99mywd) i apologise for the quality but i can't be bothered relaunching tf2 and adjusting sharex bullshit.

Basically when i turn on fresnel for enemies, my team, me and some props use same color/material. Also some weird shit appears

**Comment by xxmyn** [comment]
I have not been able to reproduce this issue with the description given, I also cannot access your config so that's cool
